modulefiles/build_hera_gnu.lua

```diff
@@ -5,6 +5,9 @@ the NOAA RDHPC machine Hera using GNU 9.2.0
 
 whatis([===[Loads libraries needed for building the UFS SRW App on Hera using GNU 9.2.0 ]===])
 
+-- When Hera switces from CentOS to Rocky, replace line withh correct path to spack-stack
```

Comment:
`switces` should be `switches`. Same goes for `switces` in line 11 of the `modulefiles/build_hera_intel.lua` file.

Reply:
Done